### PR TITLE
Fix resize

### PR DESCRIPTION
**Pull request by xukai92**

### CODE REVIEW

File: src/adaptation/precond.jl [hunk 1]
```diff
@@ -227,7 +227,7 @@ function Base.resize!(
     if length(θ) != length(dpc.var)
         @assert dpc.ve.n == 0 "Cannot resize a var estimator when it contains samples."
         dpc.ve = WelfordVar(T, length(θ))
-        dpc.var = zeros(T, length(θ))
+        dpc.var = ones(T, length(θ))
```
Comment (yebai) [inline src/adaptation/precond.jl]: nice catch!

**Comment by yebai** [comment]
Seems that some tests are broken?

**Comment by xukai92** [comment]
I guess we were testing against a vector of zeros for the function I fixed? I am not with my computer ATM. Can you check and maybe fix the test?

**Comment by yebai** [comment]
Sure - Fixed.

**Comment by xukai92** [comment]
Thanks!

**Comment by xukai92** [comment]
Shall we register it?

**Pull request merged**